lib/domain/alarm_scheduler.dart

```diff
@@ -136,8 +271,7 @@ class AlarmScheduler {
   int _getNextScheduledDay(int currentDay, List<Weekday> enabledDays) {
     // Sort enabled days by their corresponding int values for easy traversal
     final tmp = List.from(enabledDays);
-    final sortedEnabledDays = tmp
-      ..sort((a, b) => a.position.compareTo(b.position));
+    final sortedEnabledDays = tmp..sort((a, b) => a.position.compareTo(b.position));
```

Comment:
nit: Ove dve linije mozda mogu da se spoje u jednu
```suggestion
    final sortedEnabledDays = List.from(enabledDays)..sort((a, b) => a.position.compareTo(b.position));
```

Reply:
Mbbbaaa